.buildconfig.yml

```diff
@@ -348,6 +348,7 @@ projects:
     path: components/tooling/detekt
     description: 'Custom Detekt rules for internal use.'
     publish: false
+    artifact-type: jar
```

Comment:
Nit: This particular component has `publish: false` so this change won't have any effect.

Reply:
We would like to publish this too once we can publish jars. However this can also wait until someone from AC has time to test that. :)

Reply:
Ok.  Maybe I should combine this with #4985, which adds the artifact *I* care about (the Glean Gradle plugin) and then enable that rather than `tooling.detekt`?